src/app/mork/tests/celery/tasks/test_edx.py

```diff
@@ -121,7 +156,11 @@ def test_delete_edx_platform_user_invalid_status(db_session, monkeypatch):
         "mork.celery.tasks.edx.update_status_in_mork", mock_update_status_in_mork
     )
 
-    delete_edx_platform_user(user.id)
+    with pytest.raises(
+        UserStatusError,
+        match=f"User {user.id} is not to be deleted. Status: DeletionStatus.DELETED",
+    ):
+        delete_edx_platform_user(user.id)
 
     mock_delete_edx_mysql_user.assert_not_called()
     mock_delete_edx_mongo_user.assert_not_called()
```

Comment:
You could also check that the status has been updated to `Protected` in mork, WDYT ?